web-client/src/primitives/hash.rs

```diff
@@ -0,0 +1,29 @@
+use nimiq_serde::Serialize;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct Hash;
+
+#[wasm_bindgen]
+impl Hash {
+    #[wasm_bindgen(js_name = computeBlake2b)]
+    pub fn compute_blake2b(data: &[u8]) -> Vec<u8> {
+        nimiq_hash::Hasher::digest(nimiq_hash::Blake2bHasher::default(), data).serialize_to_vec()
+    }
+
+    #[wasm_bindgen(js_name = computeArgon2d)]
+    pub fn compute_argon2d(data: &[u8]) -> Vec<u8> {
```

Comment:
Could you add some small `rustdoc`?

Reply:
> Computes a 32-byte Argon2 hash. Argon2d hashes are used for example for <purpose>.

Please find out for what, @jsdanielh 😅

Reply:
We actually don't have a specific usage for it :smile:

Reply:
I think it was the PoW hash.

Reply:
You are right